src/Infrastructure.VS.UnitTests/ActiveDocumentLocatorTests.cs

```diff
@@ -41,18 +41,37 @@ public class ActiveDocumentLocatorTests
         [TestInitialize]
         public void TestInitialize()
```

Comment:
I looked at refactoring the tests to remove the `TestInitialize` but decided it would introduce too much churn in this case.